Sophos/trigger_sophos_xdr_query.json

```diff
@@ -0,0 +1,36 @@
+{
+    "description": "Forward Sophos XDR Data lake Events",
+    "docker_parameters": "sophos_query_events_trigger",
+    "name": "Get Sophos Data lake events",
+    "arguments": {
+      "$schema": "http://json-schema.org/draft-07/schema#",
+      "properties": {
+        "chunk_size": {
+          "type": "integer",
+          "description": "The max size of chunks for the batch processing",
+          "default": 1000
+        },
+        "intake_server": {
+          "description": "Server of the intake server (e.g. 'https://intake.sekoia.io')",
+          "default": "https://intake.sekoia.io",
+          "type": "string"
+        },
+        "intake_key": {
+          "description": "Intake key to use when sending events",
+          "type": "string"
+        },
+        "query": {
+          "description": "A query to get informations from Sophos data lake",
+          "type": "dict"
```

Comment:
I'm not willing to let the customer defines the query. 
I propose to keep `SophosXDRQueryTrigger` as a base class and derivates children classes with hard-coded queries. To debate.

Reply:
Hello @squioc, i have add a child class with a specific query. which can cache the time but the idea that comes in my mind is instead of caching the last time of an event it's better to cache the time where we have done the query, as we have the option of `from` in the query instead of `calendar_time > "time"`

Reply:
yes, that's it